Sources/AWSSDKSwiftCore/Serializer/XMLNodeSerializer.swift

```diff
@@ -122,10 +122,16 @@ public class XMLNodeSerializer {
             let memberChildren: [XMLNode]
 
             if arrayNodes.isEmpty {
-                jsonStr += "{"
-                memberChildren = memberNode.children.filter({ !keys.contains($0.elementName) })
-                jsonStr += _serialize(nodeTree: memberChildren)
-                jsonStr += "}"
+                if memberNode.children.isEmpty {
```

Comment:
brilliant. I was up in the `_serialize` method trying to analyze memberNode (with `.hasSingleValue()` for example) to skip jumping into `_processNodeWithMember` entirely, but this is much simpler and more elegant

Reply:
🙌 

This also makes sense as the place to edit since the previous output was getting us `{}`, showing this is definitely the right spot to be putting in the conditional.

Reply:
Yup, this gets me further along 👍